tgui/packages/tgui/styles/components/Button.scss

```diff
@@ -40,6 +40,10 @@ $bg-map: colors.$bg-map !default;
     background-color: lighten($color, 15%);
     color: $text-color;
   }
+
+  &:active {
+    background-color: lighten($color, 60%);
+  }
```

Comment:
I initially had a transition property here, but due to how fast mouse clicks actually happen (even <50ms felt slightly unresponsive) I decided to remove it.

Reply:
I'm a fan in general, and as focus does `lighten` there's precedent for this implementation so I'm fine with it. Consider making just this change as a PR to tgstation to see if it can be accepted upstream. The more our components differ from theirs the harder keeping things in sync with regards to updates is.

I would look to possibly do both usages of `lighten` (here and for `:hover` and `:focus`) as a "lighten or darken, depending on what has higher contrast" mixin (see the `$text-color` variable a little higher up for an example). If you'd like to dabble with that as part of this PR feel free (i.e. makes sense to change once if we can), no worries if not.

Reply:
I might do it as part of a separate PR